lib/orderbook/OrderBook.ts

```diff
@@ -341,14 +345,22 @@ class OrderBook extends EventEmitter {
       };
     }
 
-    if (!this.nosanitychecks && this.swaps) {
-      // check if sufficient outbound channel capacity exists
+    if (this.swaps) {
       const { outboundCurrency, outboundAmount } = Swaps.calculateInboundOutboundAmounts(order.quantity, order.price, order.isBuy, order.pairId);
       const swapClient = this.swaps.swapClientManager.get(outboundCurrency);
+
+      // check if client exists
       if (!swapClient) {
         throw swapsErrors.SWAP_CLIENT_NOT_FOUND(outboundCurrency);
       }
-      if (outboundAmount > swapClient.maximumOutboundCapacity) {
+
+      // Check if client is supported
```

Comment:
As mentioned above - we don't need this logic because swap client for currency won't exist when it is disabled.